planning/motion_velocity_planner/autoware_motion_velocity_run_out_module/README.md

```diff
@@ -162,6 +162,7 @@ The decision table can be visualized on the debug markers with the `decisions` n
 Finally, for each object, we calculate how the velocity profile will be modified based on the decision made:
 
 - `stop`: insert a `0` velocity ahead of the predicted collision point by the distance set in the `stop.distance_buffer` parameter.
+  - to prevent the stop pose from jumping around, the previous stop pose is reused if it is before the newly calcuted stop pose, but only if the arc length between the two poses is less than the `stop.reuse_margin` parameter.
```

Comment:
```suggestion
  - to prevent the stop pose from jumping around, the previous stop pose is reused if it is before the newly calculated stop pose, but only if the arc length between the two poses is less than the `stop.reuse_margin` parameter.
```